.github/workflows/comment-on-pr.yml

```diff
@@ -31,8 +31,8 @@ jobs:
              - [Writing tests](https://docs.plasmapy.org/en/latest/contributing/testing_guide.html#testing-guide) using [pytest](https://docs.pytest.org)
              - [Adding a changelog entry](https://docs.plasmapy.org/en/latest/contributing/changelog_guide.html#adding-a-changelog-entry) (needed except for minor changes)
 
-            > [!NOTE]
-            > PlasmaPy recently switched to an [\`src\`](https://packaging.python.org/en/latest/discussions/src-layout-vs-flat-layout/) layout. The source code previously in \`plasmapy/\` is now in [\`src/plasmapy/\`](https://github.com/PlasmaPy/PlasmaPy/tree/main/src/plasmapy). Tests are now in [\`tests/\`](https://github.com/PlasmaPy/PlasmaPy/tree/main/tests).
+            > [!IMPORTANT]
+            > PlasmaPy recently switched to an [\`src\`](https://packaging.python.org/en/latest/discussions/src-layout-vs-flat-layout/) layout. The source code previously in \`plasmapy/\` is now in [\`src/plasmapy/\`](https://github.com/PlasmaPy/PlasmaPy/tree/main/src/plasmapy). Tests are now in [\`tests/\`](https://github.com/PlasmaPy/PlasmaPy/tree/main/tests). If you have previously done an editable installation, it will likely need to be re-done (i.e., with `pip install -e .[tests,docs]` in the top-level directory of the repository). The former `plasmapy/` directory will need to be deleted manually in old clones because `git` does not track directories.
```

Comment:
```suggestion
            > PlasmaPy recently switched to an [\`src\`](https://packaging.python.org/en/latest/discussions/src-layout-vs-flat-layout/) layout. The source code previously in \`plasmapy/\` is now in [\`src/plasmapy/\`](https://github.com/PlasmaPy/PlasmaPy/tree/main/src/plasmapy). Tests are now in [\`tests/\`](https://github.com/PlasmaPy/PlasmaPy/tree/main/tests). If you have previously done an editable installation, it will likely need to be re-done (i.e., with `pip install -e .[tests,docs]` in the top-level directory of the repository). The former `plasmapy/` directory will need to be deleted manually in old clones because \`git\` does not track directories.
```